src/Webfactory/Slimdump/Task.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Webfactory\Slimdump;
+
+use Symfony\Component\Console\Output\OutputInterface;
+use Webfactory\Slimdump\Config\Config;
+use Webfactory\Slimdump\Config\ConfigBuilder;
+use Webfactory\Slimdump\Database\Dumper;
+use Doctrine\DBAL\Connection;
+
+class Task
```

Comment:
Do we have any reason to make it final? We don't inherit from DumpClass (as it's called now) because we don't need to, but would it cause any problems if we did?

Reply:
IMHO Classes should be final by default. This reduces your cognitive load.

There are good reasons to make a class non final - e.g. when this is your explicit extension point and composition is worse for some reason. Another controversial reason is mockability for unit tests. But I don't see any of them apply here.

I'm not sure what DumpClass in this context is, just to make sure we have a common understanding of the final keyword: 
1. A class cannot extend a final class.
2. A final class can extend another non final class.

Reply:
If final ist the default, I will make it final. I meant "DumpTask" instead of "DumpClass", sorry. That's the new name of "Task" after my refactoring.